### PR TITLE
examples: Update 34401A example to use an enum

### PR DESCRIPTION
**Pull request by bkeryan**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Update the 34401A example to use an enum instead of a string for the `measurement_type` input.

The 34401A instrument driver uses a U16 ring control for this input, so I copied it into a `.ctl`, replaced it with an NXG/fuse style enum, and converted it to a typedef.

### Why should this Pull Request be merged?

Simplify example and demonstrate proper usage.

### What testing has been done?

Manually tested.

### CODE REVIEW

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.seq
Comment (bkeryan) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.seq:1]: image

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement Configuration.ctl
Comment (bkeryan) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement Configuration.ctl:1]: image

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/subVIs/Keysight 34401A Function.ctl
Comment (bkeryan) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/subVIs/Keysight 34401A Function.ctl:1]: image

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/subVIs/Keysight 34401A Measurement Logic.vi
Comment (bkeryan) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/subVIs/Keysight 34401A Measurement Logic.vi:1]: Removed call to `Get Function.vi`

image

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement UI.vi
Comment (bkeryan) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement UI.vi:1]: image

image

**Review by dixonjoel — APPROVED** [review]

**Review by pbirkhol-ni — APPROVED** [review]
I can't easily test this, but I did open it in InstrumentStudio and it looks good.

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement UI.vi
Comment (bkeryan) [reply Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement UI.vi:1]: Compare to measUI for Python nidmm_measurement:
image

The enum's increment/decrement should not stick out to the left.

I guess we didn't line up the pin name and measurement type controls' right edges in the measui either.

Comment (pbirkhol-ni) [reply Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement UI.vi:1]: This type of issue doesn't bother me that much, but if I had to choose, I like the python UI better.

Comment (bkeryan) [reply Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement/Measurement UI.vi:1]: A little wider so "4-wire Resistance" fits.
image

**Pull request merged**